db/seeds.rb

```diff
@@ -128,3 +128,45 @@
   brewery_id: 6,
   user_id: 1
 )
+
+Beer.create(
+  name: "A Beer",
+  beer_type: "Laaaaaaager",
+  description: "Piss in a glass",
+  brewery_id: 1
+)
```

Comment:
You could have just called it Bud Light, we all know which one you're talking about ;)